tests/virt_autotest/login_console.pm

```diff
@@ -42,7 +42,7 @@ sub login_to_console {
     if (check_var('PERF_KERNEL', '1')) {
         reset_consoles;
         select_console 'sol', await_console => 0;
-        send_key_until_needlematch(['linux-login', 'virttest-displaymanager'], 'ret', $timeout, 5);
+        send_key_until_needlematch(['linux-login', 'virttest-displaymanager'], 'ret', 5, $timeout);
```

Comment:
The default timeout is 240*3 actually, do you really want such big timeout, it is a waste of time IMO for your case.

Reply:
OK. But it seems like a $timeout as $counter, also need loop many times, right ?

Reply:
Yes, it is bad. So with your change this time, i suggest you to make all work well :).  Better, isn't it?

Reply:
No, I only point out this parameter might be put in the wrong position.
A*B = B*A, the loop times should be same, whatever $timeout at 3rd or 4th position.

Reply:
I get your point. 
Difference is that, in the original way, although it is wrong, but it verifies whether it is correct screen in every 5*3 seconds, but after you change it, it will do it in every 240*3=12min. So your PR will make test less timely than before.   Do you really want this? If you want, then i can merge it.

Reply:
I don't know, might be $timeout, could be configured will it is wrong(too big).
If it could be improved, I agree close this PR, and open another. Joyce might be interested to do this.

Reply:
Hi,James,could you please change $timeout =5 and $counter=240